### ISSUE TITLE
Dont use this Library. Its Dead

### ISSUE THREAD

**Issue by GodAlmighty990** [react-bootstrap-table/react-bootstrap-table2#1774]
This library is of no use anymore. project has been abandoned.

**Comment by ayush-lal** [comment]
Are there any decent alternatives?

**Comment by GodAlmighty990** [comment]
Yes, use react-table.  You would be surprised at how similar it is to this project. There is also a great youtube series from a channel called “codevolution” where he details step by step how to use react-table. I scrapped react-bootstrap-table-next and switched to react-table and built an amazing table for my app. Just go to youtube and type in codevolution react table

**Comment by avindra** [comment]
or just `<table>...</table>` u no

**Comment by axlchr12** [comment]
does react-table supports server pagination and responsive table?
